.gitignore

```diff
@@ -1,3 +1,5 @@
+# Local files
+
```

Comment:
Please remove these changes; the only changed file should be `Grid/threads/Accelerator.h`.